packages/@o3r/application/package.json

```diff
@@ -15,16 +15,18 @@
     "prepare:publish": "dispatch-package-json-fields ./dist/package.json"
   },
   "peerDependencies": {
+    "@angular-devkit/schematics": "~15.0.4",
     "@angular/common": "^15.0.2",
     "@angular/core": "^15.0.2",
     "@o3r/core": "workspace:^",
-    "@o3r/dev-tools": "workspace:^",
     "@o3r/logger": "workspace:^",
     "@o3r/routing": "workspace:^",
     "@o3r/testing": "workspace:^",
     "rxjs": "^7.4.0"
   },
   "dependencies": {
+    "@o3r/dev-tools": "workspace:^",
```

Comment:
why is it mandatory as `dep` instead or `devDep`

Reply:
moved as dev dep